Added Dark Mode

Palakkgoyal opened the pull request:
I have added dark mode in the website. I have also used local storage so, if anyone come to website even after so many days then still the website remember their setting. The dark mode button also don't mess with different screen sizes.
![Screenshot (449)](https://user-images.githubusercontent.com/116902573/227589372-be125952-f47b-4ba2-ae8b-4d9a408035ca.png)

himanshu1221 commented:
hey @Palakkgoyal good work what I suggest is you should change the colour of the logo `linkfree` and `add your profile today` to `white` when we switch to dark background as it is mixing with it

himanshu1221 reviewed (changes requested):
hey @Palakkgoyal good work what I suggest is you should change the colour of the logo `linkfree` and `add your profile today` to `white` when we switch to dark background as it is mixing with it and please change the dark bg color to `#212121`

this is also glitchy name and bio is not visible 

![Screenshot from 2023-03-25 08-13-20](https://user-images.githubusercontent.com/32031706/227682765-fe53aebc-07ac-49d5-916a-3b32637cd387.png)

Palakkgoyal commented:
Which background are you referring to? Are you referring to the one whose color I have changed or the middle blue one?
![Screenshot (450)](https://user-images.githubusercontent.com/116902573/227700363-b0cb0997-fc4b-4709-9e6f-a9af79bf1d6a.png)

himanshu1221 commented:
> Which background are you referring to? Are you referring to the one whose color I have changed or the middle blue one? ![Screenshot (450)](https://user-images.githubusercontent.com/116902573/227700363-b0cb0997-fc4b-4709-9e6f-a9af79bf1d6a.png)

<h4>there is a route in the website which redirects you to where all links are present you can have a look here</h4>

- can you please enhance the navbar and content in the middle part as it is not looking beautiful 

`https://linkfree-clone.vercel.app/himanshu1221`

Palakkgoyal commented:
Yaa. sure

Palakkgoyal commented:
I have done some changes. Would you like to consider them and give suggestions?
![Uploading Screenshot (451).png…]()

Palakkgoyal commented:
I have done some changes. Would you like to consider them and give suggestions?
![Screenshot (451)](https://user-images.githubusercontent.com/116902573/227708190-9bbfd54d-6463-4107-8004-bcc6e5ff7d17.png)

himanshu1221 commented:
> I have done some changes. Would you like to consider them and give suggestions? ![Screenshot (451)](https://user-images.githubusercontent.com/116902573/227708190-9bbfd54d-6463-4107-8004-bcc6e5ff7d17.png)

this is looking nice :+1:

Palakkgoyal commented:
I have pushed the modifications. If you think I can improve anything else, please consider. Actually, I am not that familiar with `primereact` but, I have checked its documentation and did a change but that change was not showing up to me. Like even before when I added the darkmode icon in the navbar that was not appearing to me in the profile section. But in the screenshot you have send here it appears. So after checking tell me, if the profile is corrected or not?

> > Which background are you referring to? Are you referring to the one whose color I have changed or the middle blue one? ![Screenshot (450)](https://user-images.githubusercontent.com/116902573/227700363-b0cb0997-fc4b-4709-9e6f-a9af79bf1d6a.png)
> 
> #### there is a route in the website which redirects you to where all links are present you can have a look here
> * can you please enhance the navbar and content in the middle part as it is not looking beautiful
> 
> `https://linkfree-clone.vercel.app/himanshu1221`

Palakkgoyal commented:
I have pushed the modifications. If you think I can improve anything else, please consider. Actually, I am not that familiar with primereact but, I have checked its documentation and did a change but that change was not showing up to me. Like even before when I added the darkmode icon in the navbar that was not appearing to me in the profile section. But in the screenshot you have send here it appears. So after checking tell me, if the profile is corrected or not?

> hey @Palakkgoyal good work what I suggest is you should change the colour of the logo `linkfree` and `add your profile today` to `white` when we switch to dark background as it is mixing with it and please change the dark bg color to `#212121`
> 
> this is also glitchy name and bio is not visible
> 
> ![Screenshot from 2023-03-25 08-13-20](https://user-images.githubusercontent.com/32031706/227682765-fe53aebc-07ac-49d5-916a-3b32637cd387.png)

himanshu1221 commented:
> I have pushed the modifications. If you think I can improve anything else, please consider. Actually, I am not that familiar with primereact but, I have checked its documentation and did a change but that change was not showing up to me. Like even before when I added the darkmode icon in the navbar that was not appearing to me in the profile section. But in the screenshot you have send here it appears. So after checking tell me, if the profile is corrected or not?
> 
> > hey @Palakkgoyal good work what I suggest is you should change the colour of the logo `linkfree` and `add your profile today` to `white` when we switch to dark background as it is mixing with it and please change the dark bg color to `#212121`
> > this is also glitchy name and bio is not visible
> > ![Screenshot from 2023-03-25 08-13-20](https://user-images.githubusercontent.com/32031706/227682765-fe53aebc-07ac-49d5-916a-3b32637cd387.png)

Hey @Palakkgoyal it's looking awesome and yes switch is available in the profile section but kindly change the color of the text in profile page 
![Screenshot from 2023-03-26 21-04-21](https://user-images.githubusercontent.com/32031706/227790792-236e5f7b-1283-47e7-b1f0-7edc70fcae3a.png)

Palakkgoyal commented:
I have done a change, try that. The problem is whenever I click on the `Example` button, it take me away from the localhost and navigate to a new link. Due to this I am not able to take charge of the changes.
![Screenshot (454)](https://user-images.githubusercontent.com/116902573/228161756-2ba1ca7c-ae5a-4783-bf1d-f5147348a1cb.png)

himanshu1221 commented:
> > I have done a change, try that. The problem is whenever I click on the `Example` button, it take me away from the localhost and navigate to a new link. Due to this I am not able to take charge of the changes. ![Screenshot (454)](https://user-images.githubusercontent.com/116902573/228161756-2ba1ca7c-ae5a-4783-bf1d-f5147348a1cb.png)
> 
> Hey palak it still not working text is still black in dark mode `https://linkfree-clone-git-fork-palakkgoyal-dark-himanshu1221.vercel.app/himanshu1221` or you can navigate to `https://linkfree-clone.vercel.app/himanshu1221` you need to navigate thru link
> 
> ![Screenshot from 2023-03-28 16-37-09](https://user-images.githubusercontent.com/32031706/228217305-13314f20-1c8e-4b37-ba2c-98700f61cbbc.png)

any updates  @Palakkgoyal ?

Palakkgoyal commented:
Yaa, actually I am not aware of Routes, so I was taking some notes of it. But, still I am not able to solve. Let me try it today too. And tomorrow if you want you can assign it to someone else.
